tests/semanticdb/metac.expect

```diff
@@ -3916,148 +4071,148 @@ Symbols => 142 entries
+local13 => typeparam T 
+local14 => type L [typeparam T ] = List[T]
+types/B# => class B extends Object { self: B => +1 decls }
+types/B#`<init>`(). => primary ctor <init> (): B
+types/C# => class C extends Object { self: C => +1 decls }
+types/C#`<init>`(). => primary ctor <init> (): C
+types/Foo# => case class Foo extends Object with Product with Serializable { self: Foo => +5 decls }
+types/Foo#_1(). => method _1 => "abc"
+types/Foo#`<init>`(). => primary ctor <init> (val param s: "abc"): Foo
+types/Foo#`<init>`().(s) => val param s: "abc"
+types/Foo#copy$default$1(). => method copy$default$1 => "abc" @uncheckedVariance
+types/Foo#copy(). => method copy (param s: "abc"): Foo
+types/Foo#copy().(s) => param s: "abc"
+types/Foo#s. => val method s "abc"
+types/Foo. => final object Foo extends Object { self: Foo.type => +6 decls }
+types/Foo.apply(). => method apply (param s: "abc"): Foo
+types/Foo.apply().(s) => param s: "abc"
+types/Foo.toString(). => method toString => String
+types/Foo.unapply(). => method unapply (param x$1: Foo): Foo
+types/Foo.unapply().(x$1) => param x$1: Foo
+types/Foo.x. => val method x "abc" @deprecated
+types/Foo.y. => val method y "abc"
+types/P# => class P extends Object { self: P => +8 decls }
+types/P#C# => class C extends Object { self: C => +1 decls }
+types/P#C#`<init>`(). => primary ctor <init> (): C
+types/P#X# => class X extends Object { self: X => +1 decls }
+types/P#X#`<init>`(). => primary ctor <init> (): X
+types/P#`<init>`(). => primary ctor <init> (): P
+types/P#x. => val method x X
+types/T# => class T extends Object { self: T => +8 decls }
+types/T#C# => class C extends Object { self: C => +1 decls }
+types/T#C#`<init>`(). => primary ctor <init> (): C
+types/T#X# => class X extends Object { self: X => +1 decls }
+types/T#X#`<init>`(). => primary ctor <init> (): X
+types/T#`<init>`(). => primary ctor <init> (): T
+types/T#x. => val method x X
+types/Test. => final object Test extends Object { self: Test.type => +10 decls }
+types/Test.C# => class C extends M { self: C => +42 decls }
+types/Test.C#ByNameType. => final object ByNameType extends Object { self: ByNameType.type => +2 decls }
+types/Test.C#ByNameType.m1(). => method m1 (param x: => Int): Int
+types/Test.C#ByNameType.m1().(x) => param x: => Int
+types/Test.C#ClassInfoType1. => final object ClassInfoType1 extends Object { self: ClassInfoType1.type => +1 decls }
+types/Test.C#ClassInfoType2# => class ClassInfoType2 extends B { self: ClassInfoType2 => +2 decls }
+types/Test.C#ClassInfoType2#`<init>`(). => primary ctor <init> (): ClassInfoType2
+types/Test.C#ClassInfoType2#x(). => method x => Int
+types/Test.C#ClassInfoType3# => trait ClassInfoType3 [typeparam T ] extends Object { self: ClassInfoType3[T] => +2 decls }
+types/Test.C#ClassInfoType3#[T] => typeparam T 
+types/Test.C#ClassInfoType3#`<init>`(). => primary ctor <init> [typeparam T ](): ClassInfoType3[T]
+types/Test.C#Either. => val method Either Either.type
```

Comment:
perhaps `prefixBeforeTpe` in `PPrint.scala` could have an adjustment for when it is a Method that has `ValueSignature(tpe)` where `tpe` is not `ByNameType`